model/src/main/scala/org/broadinstitute/dsde/rawls/model/Notification.scala

```diff
@@ -73,19 +73,19 @@ object Notifications {
     override val alwaysOn = true
   })
 
-  case class WorkspaceAddedNotification(recipientUserId: String, accessLevel: String, workspaceName: WorkspaceName, workspaceOwnerEmail: String) extends UserNotification
+  case class WorkspaceAddedNotification(recipientUserId: String, accessLevel: String, workspaceName: WorkspaceName, workspaceOwnerId: String) extends UserNotification
```

Comment:
can you change these to `RawlsUserSubjectId` so it's abundantly clear these aren't emails